.gitignore

```diff
@@ -14,3 +14,6 @@ docs/.jekyll-metadata
 .ruby-version
 
 /.tmp/
+
+# asdf
+.tool-versions
```

Comment:
for my edification - is this needed for the buf linter?

Reply:
It was mostly for me. We use [asdf](https://asdf-vm.com/) to manage things like go versions between projects and I didn't want to commit the config file by accident.

Reply:
Would a global gitignore work? That will help minimize putting user-specific configuration in this repository:

https://git-scm.com/docs/gitignore

> Patterns which a user wants Git to ignore in all situations (e.g., backup or temporary files generated by the user’s editor of choice) generally go into a file specified by core.excludesFile in the user’s ~/.gitconfig. Its default value is $XDG_CONFIG_HOME/git/ignore. If $XDG_CONFIG_HOME is either not set or empty, $HOME/.config/git/ignore is used instead.

Reply:
It would not. Many of our internal repositories distributed the .tool-versions to prevent version mismatch between developers. I'll simply remove it and make the effort to not accidentally add the file. I agree that it's polluting the repo.